jplag.frontend.rust/src/main/java/de/jplag/rust/RustTokenConstants.java

```diff
@@ -0,0 +1,118 @@
+package de.jplag.rust;
+
+import de.jplag.TokenConstants;
+
+public interface RustTokenConstants extends TokenConstants {
+    int NONE = -1;
```

Comment:
This will cause problems with the core algorithm. Please only use >= 0 values. Why do you need that one?

Reply:
Oh, my bad. I should say that no token with this type is ever created, instead, it is a flag used in ParserState.Context objects to indicate that no token should be added for `{`or `}` inside of this context.

I'll move it to the ParserState.Context interface where it is actually used and causes less confusion.